CHANGELOG.md

```diff
@@ -86,6 +86,8 @@ Ref: https://keepachangelog.com/en/1.0.0/
   The information can now be accessed using the BankKeeper.
   Setting can be done using MsgSetSendEnabled as a governance proposal.
   A SendEnabled query has been added to both GRPC and CLI.
+* (appModule) Remove `Route`, `QuerierRoute` and `LegacyQuerierHandler` from AppModule Interface. 
+* (x/modules) Remove all LegacyQueries and related code from modules
```

Comment:
Nice! Next step would be to remove `sdk.Route`/`sdk.Router` and all legacy router code from baseapp. In a follow-up PR?

Reply:
Yea, I'll double check how big the change is. If not big I can do it in this PR

Reply:
Best to do it in a follow up pr as it would add quite a bit of changes.